samples/star/src/androidTest/kotlin/com/slack/circuit/star/petdetail/PetDetailTest.kt

```diff
@@ -127,4 +132,37 @@ class PetDetailTest {
       }
     }
   }
+
+  @Test
+  fun petDetail_emits_event_when_tapping_on_full_bio_button() = runTest {
+    val channel = Channel<Any>(1)
+
+    val success =
+      PetDetailScreen.State.Success(
+        url = "url",
+        photoUrls = listOf("http://some.url"),
+        photoUrlMemoryCacheKey = null,
+        name = "Baxter",
+        description = "Grumpy looking Australian Terrier",
+        tags = listOf("dog", "terrier", "male"),
+        eventSink = channel::trySend
```

Comment:
nit: why the name `trySend` for this callback? Should we just do `send`?

Reply:
I can't use `send` here because it's `suspend`-able, `trySend` isn't